README.rst

```diff
@@ -99,21 +99,21 @@ installed within the range.
 .. code-block:: python
 
     from insights import rule, make_response, run
-    from insights.parsers.installed_rpms import InstalledRpms, InstalledRpm
+    from insights.parsers import installed_rpms as rpm
 
-    lower = InstalledRpm.from_package("bash-4.4.16-1.fc27")
-    upper = InstalledRpm.from_package("bash-4.4.22-1.fc27")
+    lower = rpm.Rpm("bash-4.4.16-1.fc27")
+    upper = rpm.Rpm("bash-4.4.22-1.fc27")
 
-    @rule(InstalledRpms)
+    @rule(rpm.Installed)
     def report(rpms):
-        rpm = rpms.get_max("bash")
-        if rpm and rpm >= lower and rpm < upper:
-            return make_response("BASH_AFFECTED", version=rpm.nvr)
-        elif rpm:
-            return make_response("BASH_UNAFFECTED", version=rpm.nvr)
-        else:
+        bash = rpms.get_max("bash")
+        if not bash:
             return make_response("NO_BASH_FOUND")
 
+        msg = "BASH_AFFECTED" if lower <= bash < upper else "BASH_UNAFFECTED"
```

Comment:
It seems unusual to me to return a response at all when bash is unaffected.

Reply:
I went back and forth on this.  From a support standalone script, it seemed natural to give the "green light."   It certainly is unusual from a usual plugin.  And, because of that, we may want to leave it off here to help set the idiom.

Reply:
My opinion is to leave it off for two reasons:

1. As you said, it's unnatural for a "standard" plugin
2. It make the example simpler to digest.

We can always explain how to make a diagnostic response elsewhere.

Reply:
Ack, @jhjaggars feel free to simplify the pluging stuff to drop the  `BASH_UNAFFECTED` or even `BASH_NOT_FOUND`.